webapp/stockstream/player.py

```diff
@@ -1,12 +1,17 @@
 import stockstream
 import robinhood
-import json
 
-from operator import itemgetter
+
+def compute_change_decimal(from_value, to_value):
+    if from_value == 0 and to_value == 0:
+        return 0
+    from_value = 1 if from_value == 0 else from_value
```

Comment:
Maybe `from_value = min(1, from_value)` would be a little clearer?

Reply:
Good suggestion- though it's actually max(1, from_value) since it's really just setting to 1 if 0 and value otherwise.